{
  "original_file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\risk_management\\live_risk_governor_recovered_2.py",
  "quarantine_timestamp": "2025-06-21T04:33:30.244634+00:00",
  "violations": [
    {
      "rule_type": "no_fallback_logic",
      "pattern": "try:",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\risk_management\\live_risk_governor_recovered_2.py",
      "line_numbers": [
        190,
        217,
        239,
        307,
        380
      ],
      "violation_count": 5
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "except Exception",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\risk_management\\live_risk_governor_recovered_2.py",
      "line_numbers": [
        211,
        234,
        267,
        338,
        410
      ],
      "violation_count": 5
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "if not",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\risk_management\\live_risk_governor_recovered_2.py",
      "line_numbers": [
        276,
        382
      ],
      "violation_count": 2
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "else:",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\risk_management\\live_risk_governor_recovered_2.py",
      "line_numbers": [
        253
      ],
      "violation_count": 1
    },
    {
      "rule_type": "eventbus_required",
      "pattern": "emit(",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\risk_management\\live_risk_governor_recovered_2.py",
      "line_numbers": [
        106,
        172
      ],
      "violation_count": 2
    }
  ],
  "quarantine_reason": "ARCHITECT_MODE_VIOLATION"
}